### PR TITLE
lapin evaluation filter

### PR DESCRIPTION
**Pull request by LnL7**

### CODE REVIEW

**Review by cole-h — APPROVED** [review]

File: ofborg/src/bin/evaluation-filter.rs [hunk 1]
```diff
@@ -1,90 +1,77 @@
...
-            auto_delete: false,
-            no_wait: false,
-        })
-        .unwrap();
+    chan.bind_queue(easyamqp::BindQueueConfig {
+        queue: queue_name.clone(),
+        exchange: "github-events".to_owned(),
+        routing_key: Some("pull_request.nixos/nixpkgs".to_owned()),
+        no_wait: false,
+    })?;
```
Comment (grahamc) [inline ofborg/src/bin/evaluation-filter.rs:57]: This change is binding `mass-rebuild-check-jobs` to subscribe directly to `github-events`, where `mass-rebuild-check-inputs` should be binding to `github-events` and then this task writes the filtered tasks to the jobs queue.

Comment (cole-h) [reply ofborg/src/bin/evaluation-filter.rs:57]: Good catch, sorry I missed that.

**Comment by grahamc** [comment]
No worries, it happens to the best of us. Accidents will happen and things will break sometimes. The benefits of this PR and making progress in general far outweigh the risks!

Thank you for the PR and for the review, keep up the good work :)


On Fri, May 22, 2020, at 11:04 AM, Cole Helbling wrote:
> 

> *@cole-h* commented on this pull request.

> 
> In ofborg/src/bin/evaluation-filter.rs <https://github.com/NixOS/ofborg/pull/491#discussion_r429301034>:

> > +    chan.bind_queue(easyamqp::BindQueueConfig {
+        queue: queue_name.clone(),
+        exchange: "github-events".to_owned(),
+        routing_key: Some("pull_request.nixos/nixpkgs".to_owned()),
+        no_wait: false,
+    })?;

File: ofborg/src/bin/evaluation-filter.rs (hunk 1 above)
Comment (LnL7) [reply ofborg/src/bin/evaluation-filter.rs:57]: These kind of subtile bugs are also really hard to catch during review if you're not looking for something specific.

**Pull request merged**